templates/community-viz/README.md

```diff
@@ -25,38 +25,38 @@ directory.
 
 ## Using this template
 
-To create a new community visualization using this template, run the command
+To create a new community visualization based on this template, run the command
 
 ```bash
 npx @google/dscc-gen --project_choice community-viz
 ```
 
 At the end of the setup flow, you will have a minimal working visualization and
-have set GCS buckets for a dev and prod version.
+have set GCS buckets for a dev and prod version. Edit `src/index.js` to see changes.
 
-### Writing the viz
+### Update your local data
 
-1. Define the dimensions and metrics your visualization requires in
+1. Update the dimensions and metrics your visualization requires in
    `src/index.json`
-2. Run the commands `npm run build:dev and; npm run push:dev` to build and
+1. Run the commands `npm run updateMessage -- --format object` to build and
```

Comment:
Since this is in the template directory, we can make this say "yarn .." or "npm ..." depending on choice, defaulting to npm. Could you add an issue to address this once this PR is merged in?